src/components/home/summary/item.vue

```diff
@@ -0,0 +1,86 @@
+<!--
+Copyright 2022 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <linkable :to="to" class="home-summary-item">
+    <div class="heading">
+      <span :class="`icon-${icon}`"></span>
+      <div>
+        <div v-if="count != null" class="count">{{ $n(count, 'default') }}</div>
+        <div class="title">
+          <span><slot name="title"></slot></span>
+          <span v-if="to != null" class="icon-angle-right"></span>
```

Comment:
If there isn't this icon, there is too little space between the count and the title. Ideas:

- Increase the line height of `.title` so that it always matches what it would be with the icon?
- Make the icon transparent?